.changes/2.3.0.md

```diff
@@ -3,6 +3,7 @@
 NOTES:
 
 * Provider has been re-written using the new [`terraform-plugin-framework`](https://www.terraform.io/plugin/framework) ([#170](https://github.com/hashicorp/terraform-provider-archive/pull/170)).
+* resource/archive_file: The `archive_file` resource was deprecated in [v2.3.0](https://github.com/hashicorp/terraform-provider-archive/releases/tag/v2.3.0) and will be removed in the next major release.
```

Comment:
I _believe_ we actually don't need to explicitly note this because the terraform-plugin-sdk [`schema.DataSourceResourceShim` behavior](https://github.com/hashicorp/terraform-plugin-sdk/blob/1798ae4ae71f7ffff29312a07c92d41205aff03b/helper/schema/data_source_resource_shim.go#L21-L41) was to include a deprecation message already in the schema. From a product perspective it may actually make sense for us to plan on fully implementing the managed resource (handling deletion, etc) in the future, rather than removing it.

```suggestion
```